_posts/2022-11-09-OpenSearch-Partner-Highlight-Using-Cloaked-Search.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+title:  "OpenSearch partner highlight: Using Cloaked Search to protect your data"
+authors:
+  - zmre
+date:   2022-11-09 1
```

Comment:
apologies @BobWall23 - one last change - can you update this to today's date? We want to make sure it lists as the latest on the blog

2022-11-14

Reply:
@krisfreedain Meant to do that and forgot.

Reply:
thanks @BobWall23 !